angularSTEP/src/app/view-profiles/view-profiles.component.html

```diff
@@ -1,9 +1,9 @@
-<ais-instantsearch [config]="config">
+<!-- <ais-instantsearch [config]="config">
```

Comment:
Do you intend to delete this file?

Reply:
Yes I did. I got rid of the autocomplete functionality.

Reply:
will it be deleted when you merge this file?

Reply:
Yes

Reply:
This file appears to me like it will simply all get commented rather than being deleted. If you want to delete it then do that! No point leaving it hanging around fully commented out

Reply:
Oh I understand. Yes I will delete it.